.renovaterc.json

```diff
@@ -31,6 +31,87 @@
       "datasourceTemplate": "bitbucket-tags",
       "versioningTemplate": "regex:(?<major>\\d+).(?<minor>\\d+).(?<build>\\d+).(?<revision>\\d+)"
     },
+    {
+      "customType": "regex",
+      "description": "Update XNAT-sync plugin version specfied in roles/xnat/defaults/main.yml",
+      "fileMatch": ["roles/xnat/defaults/main.yml$"],
+      "matchStrings": ["xsync-plugin-all-(?<currentValue>.*?).jar"],
+      "depNameTemplate": "xnatdev/xsync",
+      "datasourceTemplate": "bitbucket-tags"
+    },
+    {
+      "customType": "regex",
+      "description": "Update XNAT ldap-auth plugin version specfied in roles/xnat/defaults/main.yml",
+      "fileMatch": ["roles/xnat/defaults/main.yml$"],
+      "matchStrings": ["ldap-auth-plugin-(?<currentValue>.*?).jar"],
+      "depNameTemplate": "xnatx/ldap-auth-plugin",
+      "datasourceTemplate": "bitbucket-tags"
+    },
+    {
+      "customType": "regex",
+      "description": "Update XNAT Container Service plugin version specfied in roles/xnat/defaults/main.yml",
+      "fileMatch": ["roles/xnat/defaults/main.yml$"],
+      "matchStrings": ["container-service-(?<currentValue>.*?)-fat.jar"],
+      "depNameTemplate": "xnatdev/container-service",
+      "datasourceTemplate": "bitbucket-tags"
+    },
+    {
+      "customType": "regex",
+      "description": "Update XNAT batch launch plugin version specfied in roles/xnat/defaults/main.yml",
+      "fileMatch": ["roles/xnat/defaults/main.yml$"],
+      "matchStrings": ["batch-launch-(?<currentValue>.*?).jar"],
+      "depNameTemplate": "xnatx/xnatx-batch-launch-plugin",
+      "datasourceTemplate": "bitbucket-tags"
+    },
+    {
+      "customType": "regex",
+      "description": "Update XNAT dax plugin specfied in roles/xnat/defaults/main.yml",
+      "fileMatch": ["roles/xnat/defaults/main.yml$"],
+      "matchStrings": ["dax-plugin-genProcData-(?<currentValue>.*?).jar"],
+      "depNameTemplate": "VUIIS/dax",
+      "datasourceTemplate": "github-releases"
+    },
```

Comment:
This won't work because we're [downloading a specific file from a repo](https://github.com/UCL-MIRSG/ansible-collection-xnat/blob/6b9ec3b7a8a06e002c7afa83a81e88ddfcc32b90/roles/xnat/defaults/main.yml#L46) rather than targeting a release. Any ideas how to handle this?

Reply:
I think that needs a custom datasource - https://docs.renovatebot.com/modules/datasource/custom/